[2025-06-22T16:48:20.132+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-22T16:48:20.149+0000] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: deepai_crawler_10min.crawl_hardware manual__2025-06-22T10:46:51.759159+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2025-06-22T16:48:20.152+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
